bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -22,6 +22,23 @@ spec:
         sleep 1
       done
   containers:
+  - name: cluster-policy-controller
+    image: ${CLUSTER_POLICY_CONTROLLER_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["cluster-policy-controller", "start"]
+    args:
+    - --config=/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml
+    - -v=2
+    resources:
+      requests:
+        memory: 100Mi
+        cpu: 100m
+    ports:
```

Comment:
you must have health checks before merge

Reply:
added